metricbeat/module/kubernetes/apiserver/apiserver.go

```diff
@@ -22,48 +22,46 @@ import (
 	"github.com/elastic/beats/v7/metricbeat/mb"
 )
 
-func init() {
-	//mapping := &prometheus.MetricsMapping{
-	mapping := &prometheus.MetricsMapping{
-		Metrics: map[string]prometheus.MetricMap{
-			"process_cpu_seconds_total":               prometheus.Metric("process.cpu.sec"),
-			"process_resident_memory_bytes":           prometheus.Metric("process.memory.resident.bytes"),
-			"process_virtual_memory_bytes":            prometheus.Metric("process.memory.virtual.bytes"),
-			"process_open_fds":                        prometheus.Metric("process.fds.open.count"),
-			"process_start_time_seconds":              prometheus.Metric("process.started.sec"),
-			"apiserver_watch_events_sizes":            prometheus.Metric("watch.events.size.bytes"),
-			"apiserver_response_sizes":                prometheus.Metric("response.size.bytes"),
-			"apiserver_request_duration_seconds":      prometheus.Metric("request.duration.us", prometheus.OpMultiplyBuckets(1000000)),
-			"apiserver_request_total":                 prometheus.Metric("request.count"),
-			"apiserver_current_inflight_requests":     prometheus.Metric("request.current.count"),
-			"apiserver_longrunning_gauge":             prometheus.Metric("request.longrunning.count"),
-			"apiserver_storage_objects":               prometheus.Metric("etcd.object.count"),
-			"apiserver_audit_event_total":             prometheus.Metric("audit.event.count"),
-			"apiserver_audit_requests_rejected_total": prometheus.Metric("audit.rejected.count"),
-			"rest_client_requests_total":              prometheus.Metric("client.request.count"),
-		},
+var mapping = &prometheus.MetricsMapping{
```

Comment:
As long as you correctly moved them out just you can consider use the bulk var definition as we have them here: https://github.com/elastic/beats/blob/main/metricbeat/module/kubernetes/state_pod/state_pod.go#L36

Just to be the same.